libafl_qemu/src/emu.rs

```diff
@@ -14,8 +14,9 @@ use std::{
     ffi::{CStr, CString},
     ptr::null_mut,
 };
-use std::{slice::from_raw_parts, str::from_utf8_unchecked};
+use std::{    sync::Mutex, slice::from_raw_parts, str::from_utf8_unchecked};
 
+use lazy_static::lazy_static;
```

Comment:
Instead of `lazy_static` we could use something like `OnceLock` from the rust stdlib and get around the extra dependency

Reply:
Sounds good. I'll adapt the code and push it alongside the other fixes.